client/state/data-layer/wpcom-http/actions.js

```diff
@@ -17,10 +35,11 @@ import { WPCOM_HTTP_REQUEST } from 'state/action-types';
  * @param {Object} [onProgress] Redux action to call on progress events from an upload
  * @param {Object} [options] extra options to send to the middleware, e.g. retry policy or offline policy
  * @param {Object} [action] default action to call on HTTP events
- * @returns {Object} Redux action describing WordPress.com API HTTP request
+ * @return {Object} Redux action describing WordPress.com API HTTP request
```

Comment:
okay, let's just leave it in then as-is if you didn't have a reason it was wrong. if we're going to make a purely stylistic change we can do it all at once in one PR which doesn't change behaviors.